pipelines/pipelines/eclipse/build-branch.yaml

```diff
@@ -287,8 +287,8 @@ spec:
 # 
 # 
 # 
-  # Trigger Isolated (when doing any kind of build)
-  - name: trigger-isolated
+  # Trigger CLI (when doing any kind of build)
+  - name: trigger-cli
     taskRef:
       name: tkn-cli
     runAfter:
```

Comment:
So I'd change this to Trigger CLI when doing a release build only, so change comment and add a when clause.

If I made changes to the Eclipse repo and that set off a main build, I'd only want Isolated to build after that so it has the latest Eclipse stuff. We only care about bringing the CLI in here if release.